docs/creating-new/create/cmake.rst

```diff
@@ -381,10 +381,10 @@ Substitute ``unsorted`` with some tag in directive
 
 .. note::
 
-  Since you don't know the pull request number a priori leave it as ``N`` for now.
+  Since you don't know a priory pull request number leave it as ``N`` for now.
```

Comment:
I guess you need to rebase, see https://github.com/ruslo/hunter/pull/1071.

Reply:
Also do we need pull request number, what do you think? See my comment: https://github.com/ruslo/hunter/pull/1070#discussion_r142880694

Reply:
I will rebase

I think we should add the PR, then one can see since when the project is part of hunter (with relase and date)

Reply:
I was editing the file directly though github while you merged my previous change #1071 

It seems, that when editing the new branch is based on the current master, not on the commit the edit was started on